### PR TITLE
Added option to reset counters if motion count is applied

### PR DESCRIPTION
**Pull request by phux**
`jj<oh_fuck_i_forgot>5j` works without blocking :)

### CODE REVIEW

File: plugin/hardtime.vim [hunk 1]
```diff
@@ -18,6 +18,7 @@ call s:check_defined("g:list_of_visual_keys", ["h", "j", "k", "l", "-", "+", "<U
 call s:check_defined("g:list_of_normal_keys", ["h", "j", "k", "l", "-", "+", "<UP>", "<DOWN>", "<LEFT>", "<RIGHT>"])
 call s:check_defined("g:list_of_insert_keys", ["<UP>", "<DOWN>", "<LEFT>", "<RIGHT>"])
 call s:check_defined("g:list_of_disabled_keys", [])
+call s:check_defined("g:list_of_resetting_keys", ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'])
```
Comment (takac) [inline plugin/hardtime.vim]: I don't want this as the default. `j5j` is a mistake that you should avoid.

Comment (phux) [inline plugin/hardtime.vim]: I don't consider the `5j` as the mistake. The first `j` is the mistake (not doing `6j` in the first place).
The `5j` afterwards is just applying what's actually proposed by this plugin.

But I am fine with making it optional.

**Comment by UnkwUsr** [comment]
+1 for this PR. @takac, any chance to get this merged?

File: plugin/hardtime.vim (hunk 1 above)
Comment (schneiderfelipe) [reply plugin/hardtime.vim:21]: @takac @phux I'm quite happy to have this under a flag. Would that be an option?

**Pull request merged**